gprofiler/utils.py

```diff
@@ -74,8 +75,34 @@ def get_process_nspid(pid: int) -> Optional[int]:
     # from the listing of those files itself.
     return None
 
-
-def start_process(cmd: Union[str, List[str]], via_staticx: bool, **kwargs) -> Popen:
+libc: Optional[ctypes.CDLL] = None
+def prctl(*argv):
+    global libc
+    if libc is None:
+        libc = ctypes.CDLL("libc.so.6", use_errno=True)
+    return libc.prctl(*argv)
+
+PR_SET_PDEATHSIG = 1
```

Comment:
Doesn't pass linting - please run `./enable_hook.sh` in the repo's root. It will install the commit hook that runs the linting & formatting. Then, should you try to commit again, it will make all necessary format changes (and yell at you on the lint issues it cannot fix itself). You can also run `./lint.sh` manually.